Seems to do nothing (crda not found)

Installing the plugin, it adds a new button (and context menu entry) which both seem to do nothing. It is possible to click the button, but nothing happens. No error, no information, nothing.

Taking a look at the IDE logs, I can see the following:

```
2023-01-11 11:46:21,136 [ 683350]   INFO - #org.jboss.tools.intellij.stackanalysis.SaProcessExecutor - Starting Stack Analysis.
2023-01-11 11:46:21,137 [ 683351]   INFO - #org.jboss.tools.intellij.stackanalysis.SaProcessExecutor - Authenticating user.
2023-01-11 11:46:21,139 [ 683353]   WARN - #org.jboss.tools.intellij.stackanalysis.SaAction - Cannot run program "./crda" (in directory "/home/jreimann/.local/share/JetBrains/Toolbox/apps/IDEA-U/ch-0/223.8214.52.plugins/org.jboss.tools.intellij.analytics"): error=2, No such file or directory
java.io.IOException: Cannot run program "./crda" (in directory "/home/jreimann/.local/share/JetBrains/Toolbox/apps/IDEA-U/ch-0/223.8214.52.plugins/org.jboss.tools.intellij.analytics"): error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.jboss.tools.intellij.stackanalysis.SaProcessExecutor.execute(SaProcessExecutor.java:96)
	at org.jboss.tools.intellij.stackanalysis.SaProcessExecutor.authenticateUser(SaProcessExecutor.java:50)
	at org.jboss.tools.intellij.stackanalysis.SaProcessExecutor.performStackAnalysis(SaProcessExecutor.java:71)
	at org.jboss.tools.intellij.stackanalysis.SaUtils.getReport(SaUtils.java:49)
	at org.jboss.tools.intellij.stackanalysis.SaAction.actionPerformed(SaAction.java:45)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:323)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:302)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:346)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:302)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$performAction$5(ActionMenuItem.java:296)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:226)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction(ActionMenuItem.java:289)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$new$0(ActionMenuItem.java:64)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$4(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:526)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:558)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6648)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3392)
	at java.desktop/java.awt.Component.processEvent(Component.java:6413)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2802)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:754)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:752)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:751)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:831)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:753)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:772)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:813)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:314)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:244)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 62 more
```

My expectation would be:

* The action does something (the `crda` file should be provided by the plugin, as it is not mentioned that this should be installed manually.
* In case something goes wrong, there should be an error message

--- a/src/main/java/org/jboss/tools/intellij/stackanalysis/PreloadCli.java
+++ b/src/main/java/org/jboss/tools/intellij/stackanalysis/PreloadCli.java
@@ -14,6 +14,7 @@ import com.intellij.openapi.application.ApplicationManager;
 import com.intellij.openapi.application.PreloadingActivity;
 import com.intellij.openapi.components.ServiceManager;
 import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.progress.ProcessCanceledException;
 import com.intellij.openapi.progress.ProgressIndicator;
 import org.jboss.tools.intellij.analytics.GitHubReleaseDownloader;
 import org.jboss.tools.intellij.analytics.ICookie;
@@ -66,6 +67,7 @@ public final class PreloadCli extends PreloadingActivity {
       new SaProcessExecutor().authenticateUser();
     } catch(IOException | InterruptedException e) {
       logger.warn(e);
+      throw new ProcessCanceledException(e);
     }
   }
 }

--- a/src/main/java/org/jboss/tools/intellij/stackanalysis/SaAction.java
+++ b/src/main/java/org/jboss/tools/intellij/stackanalysis/SaAction.java
@@ -18,6 +18,7 @@ import com.intellij.openapi.actionSystem.PlatformDataKeys;
 import com.intellij.openapi.actionSystem.CommonDataKeys;
 import com.intellij.openapi.diagnostic.Logger;
 import com.intellij.openapi.fileEditor.FileEditorManager;
+import com.intellij.openapi.ui.Messages;
 import com.intellij.openapi.vfs.VirtualFile;
 import com.intellij.psi.PsiFile;
 
@@ -57,6 +58,9 @@ public class SaAction extends AnAction {
             saUtils.openCustomEditor(FileEditorManager.getInstance(event.getProject()), manifestDetails);
         } catch (Exception e) {
             logger.warn(e);
+            Messages.showErrorDialog(event.getProject(),
+                    "Can't run report generation " + e.getLocalizedMessage(),
+                    "Error");
         }
     }
 

--- a/src/main/java/org/jboss/tools/intellij/stackanalysis/SaProcessExecutor.java
+++ b/src/main/java/org/jboss/tools/intellij/stackanalysis/SaProcessExecutor.java
@@ -29,7 +29,7 @@ public class SaProcessExecutor {
 	private static final Logger logger = Logger.getInstance(SaProcessExecutor.class);
 	private static final String CLI_CONFIG_FILE_PATH = Paths.get(System.getProperty("user.home"),
 			".crda", "config.yaml").toString();
-	private static final String CLI_COMMAND =  SystemInfo.isLinux ? "./crda" : "crda";
+	private static final String CLI_COMMAND =  Platform.pluginDirectory + File.separator + "crda";
 
 
 	/**

